addons/survey/models/survey_user_input.py

```diff
@@ -187,6 +198,22 @@ def action_print_answers(self):
             'url': '/survey/print/%s?answer_token=%s' % (self.survey_id.access_token, self.access_token)
         }
 
+    def action_redirect_to_attempts(self):
+        self.ensure_one()
+
+        action = self.env['ir.actions.act_window']._for_xml_id('survey.action_survey_user_input')
+        context = dict(self.env.context or {})
+
+        context['search_default_survey_id'] = self.survey_id.id
```

Comment:
Context or domain ? As we come from a form view, would makes sense to have a domain instead of search filters. Just proposing.

Reply:
Ok then.